cves/kernel/CVE-2014-7283.yml

```diff
@@ -167,8 +172,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: The issue is not something that is tested with a fuzzer.
```

Comment:
Could it have been tested by other automated tools? It doesn't necessarily have to be a fuzzer.

Reply:
I have adjusted the language used here 👍